src/components/molecules/Table/Table.jsx

```diff
@@ -56,24 +50,19 @@ const Table = () => {
               <StyledTh>&nbsp;</StyledTh>
             </tr>
             {projects.map(({ name, description, createdAt, id }) => (
-              <>
-                <tr key={id}>
-                  <StyledTd>{name}</StyledTd>
-                  <StyledTd>{description}</StyledTd>
-                  <StyledTd>{new Date(createdAt).toLocaleDateString("RU-ru")}</StyledTd>
-                  <StyledTd>
-                    <Button label="Tasks" onClick={() => routeChange(id)} color="thistle" />
-                  </StyledTd>
-                  <StyledTd>
-                    <Button label="Edit" onClick={test} color="thistle" />
-                  </StyledTd>
-                  <StyledTd>
-                    <Button label="Destroy" onClick={() => setDestroyModalOpen(true)} color="thistle" />
-                  </StyledTd>
-                </tr>
-                {isDestroyModalOpen && <ModalWindow isOpen={isDestroyModalOpen} setIsOpen={setDestroyModalOpen} />}
-              </>
+              <TableTr key={id} name={name} description={description} createdAt={createdAt} id={id} />
             ))}
+            <ModalWindowCreate isOpen={isCreateModalOpen} setIsOpen={setCreateModalOpen} />
+            <tr>
+              <StyledThLast>
+                <Button label="Create project" onClick={() => setCreateModalOpen(true)} />
+              </StyledThLast>
+              <StyledThLast> </StyledThLast>
+              <StyledThLast> </StyledThLast>
+              <StyledThLast> </StyledThLast>
+              <StyledThLast> </StyledThLast>
+              <StyledThLast> </StyledThLast>
+            </tr>
```

Comment:
in perfect, action button shouldn't be a part of a table, may be something like:
but styles should be fixed with these changes)
```
    <>
      {!loading && (
        <StyledTable>
          ...
        </StyledTable>
      )}
      {loading && <h1>LOADING NOW</h1>}
      <Button label="Create project" onClick={() => setCreateModalOpen(true)} />
    </>
```